### PR TITLE
Julia wasn't version 1.0 in 2012

### PR DESCRIPTION
**Pull request by mbauman**
Ok, I know editing this initial blog post should be taken with great care, but could we please retroactively change the wording around "releasing 1.0"? The Julia source was [version 0](https://github.com/JuliaLang/julia/commits/fee8e6dccfaf4edde5f5efdded5730ccc761ac7f/VERSION) until almost exactly a year later when it was bumped to 0.1.

### CODE REVIEW

**Comment by StefanKarpinski** [comment]
This does probably reduce confusion. Maybe add a footnote indicating what was originally written?

File: blog/2012/02/why-we-created-julia.md [hunk 1]
```diff
@@ -79,3 +79,5 @@ trackbacks:
 - [Simplified Chinese translation](https://sd.csdn.net/a/20120223/312315.html)
 - [Traditional Chinese translation](http://www.hellogcc.org/?p=20)
 - [Linuxfr.org blog post (French)](https://linuxfr.org/news/version-1-0-de-julia)
+
+[^1]: This was originally written as being "time for a 1.0 release." Version 1.0 wasn't released until [much later](https://julialang.org/blog/2018/08/one-point-zero/).
```
Comment (mbauman) [inline blog/2012/02/why-we-created-julia.md:83]: I wanted to say something snarky like:

> This was originally written as being "time for a 1.0 release," but that turned out to be overly greedy.

but I decided to keep it to the facts.

Comment (StefanKarpinski) [reply blog/2012/02/why-we-created-julia.md:83]: I kind of like the snarky version. No one minds a little self-deprecating snark :)

Comment (mbauman) [reply blog/2012/02/why-we-created-julia.md:83]:
```suggestion
[^1]: This was originally written as being "time for a 1.0 release," but that turned out to be overly greedy. Version 1.0 wasn't released until [much later](https://julialang.org/blog/2018/08/one-point-zero/).
```

Comment (StefanKarpinski) [reply blog/2012/02/why-we-created-julia.md:83]:
```suggestion
[^1]: This was originally written as being "time for a 1.0 release" but it turns out this was a bit too greedy: Julia 1.0 wasn't released until [considerably later](https://julialang.org/blog/2018/08/one-point-zero/).
```

**Comment by ViralBShah** [comment]
This seems to have added back some links I deleted yesterday that were broken, at the bottom.

**Comment by logankilpatrick** [comment]
Yeah, my bad, they were merge conflicts so it was weird. I’ll test them all and remove the ones that don’t work.

**Comment by ViralBShah** [comment]
You can see my commit to see which ones I removed...

**Comment by ViralBShah** [comment]
Or the Chinese translation.

**Pull request merged**